src/app/clusters/tls-client-management-server/app_config_dependent_sources.cmake

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is the equivalent to app_config_dependent_sources.gni
+TARGET_SOURCES(
+        ${APP_TARGET}
+        PRIVATE
+        "${CHIP_APP_BASE_DIR}/clusters/tls-certificate-management-server/CertificateTableImpl.cpp"
+        "${CHIP_APP_BASE_DIR}/clusters/tls-certificate-management-server/CertificateTableImpl.h"
```

Comment:
Please file an issue about the cmake file naming not matching how it actually behaves?

Please talk to @andy31415 about him fixing this.  When this structure landed the intent was for the cmake bits here to only have actual app-config-dependent stuff, so we need somewhere else to put the other things.

Reply:
Will chat with him, it may even be me who ends up fixing; filed https://github.com/project-chip/connectedhomeip/issues/39743